index.js

```diff
@@ -0,0 +1,16 @@
+let Merger = require('./lib/merger');
+
+let merge = (file) => {
+    return new Promise(async (resolve, reject) => {
+        let merger = new Merger({ delimeter: '\n\n' });
+        let result;
+        try {
+            result = await merger.processFile(file, true);
+        } catch (e) {
+            reject(e);
+        }
+        resolve(result);
+    });
+};
+
+module.exports = merge;
```

Comment:
We should also export `Merger` then if you need some more options to set up.

```suggestion
module.exports.Merger = Merger;
module.exports.merge = merge;
```